deepchem/models/keras_model.py

```diff
@@ -133,6 +136,8 @@ def __init__(self,
                tensorboard: bool = False,
                wandb: bool = False,
                log_frequency: int = 100,
+               logging_strategy: Optional[str] = "step",
```

Comment:
This argument needs to be documented.  The documentation for `log_frequency` also needs to change.